src/vma/dev/qp_mgr.h

```diff
@@ -147,6 +151,8 @@ class qp_mgr
 	uint32_t 		m_rx_num_wr;
 	uint32_t 		m_tx_num_wr;
 
+	bool  			m_hw_dummy_send_support;
```

Comment:
Assuming ibv_dev_ctx... will check dummy capability and passes it to ring then m_hw_dummy_send_support can be const and the check will only be in device creation and not per ring creation